tests/docs/test_docs.py

```diff
@@ -13,11 +13,13 @@ def test_html(self):
         doctrees_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['doctrees'])
         html_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['html'])
 
-        check = subprocess.call(["sphinx-build", "-nW", "-b", "html", "-d",
-            "{0!s}".format((doctrees_path)) ,
-            "{0!s}".format((self.path_to_docs)),
-            "{0!s}".format((html_path))])
-        assert check == 0
+        check = subprocess.Popen(("sphinx-build", "-nW", "-b", "html", "-d",
+            "{0!s}".format(doctrees_path),
+            "{0!s}".format(self.path_to_docs),
+            "{0!s}".format(html_path)),
+            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = check.communicate()
+        assert check == 0, 'Test failed with stdout:\n{}\n and stderr:\n{}\n'.format(out,err)
```

Comment:
both fail. This is the log for the first example (https://travis-ci.org/simpeg/simpeg/jobs/161787181) this is the second (https://travis-ci.org/simpeg/simpeg/jobs/161414352, or similarly from em.geosci: https://travis-ci.org/ubcgif/em/builds/124879386) .